apps/nextjs/src/app/[locale]/manage/layout.tsx

```diff
@@ -95,27 +98,32 @@ export default async function ManageLayout({ children }: PropsWithChildren) {
     {
       label: t("items.tools.label"),
       icon: IconTool,
-      hidden: !session?.user.permissions.includes("admin"),
+      // As permissions always include there children permissions, we can check for the lowest permission
```

Comment:
I understand what you mean but "low" is in my opinion a poor word choice to describe it (eg. what is high?)